### PR TITLE
Added default volumes and JWT environment variables

### PR DESCRIPTION
**Pull request by Vj-L**
### Type of change :

- [x] Bug fix

I have worked on providing the default volumes for each of OM services and also added the. JWT configurations in docker-compose file

### CODE REVIEW

File: docker/local-metadata/docker-compose.yml [hunk 1]
```diff
@@ -114,6 +111,11 @@ services:
       # Airflow SSL Configurations
       AIRFLOW_VERIFY_SSL: ${AIRFLOW_VERIFY_SSL:-"no-ssl"}
       AIRFLOW_SSL_CERT_PATH: ${AIRFLOW_SSL_CERT_PATH:-""}
+      # JWT Configurations
+      RSA_PUBLIC_KEY_FILE_PATH: ${RSA_PUBLIC_KEY_FILE_PATH:-""}
+      RSA_PRIVATE_KEY_FILE_PATH: ${RSA_PRIVATE_KEY_FILE_PATH:-""}
+      JWT_ISSUER: ${JWT_ISSUER:-"open-metadata.org"}
+      JWT_KEY_ID: ${JWT_KEY_ID:-"Gb389a-9f76-gdjs-a92j-0242bk94356"}
```
Comment (akash-jain-10) [inline docker/local-metadata/docker-compose.yml:118]: Is `OM_AUTH_JWT_TOKEN: ${OM_AUTH_JWT_TOKEN:-""}` missing here ?

File: docker/local-metadata/docker-compose.yml [hunk 2]
```diff
@@ -30,6 +25,8 @@ services:
       - 3306
     ports:
       - "3306:3306"
+    volumes:
+     - $PWD/docker-volume/db-data:/var/lib/mysql
```
Comment (akash-jain-10) [inline docker/local-metadata/docker-compose.yml:29]: Can you verify if the Github Actions CI dependent on this compose file will work as expected ?

File: docker/local-metadata/docker-compose-postgres.yml [hunk 3]
```diff
@@ -31,6 +26,8 @@ services:
       - 5432
     ports:
       - "5432:5432"
+    volumes:
+      - $PWD/docker-volume/db-data:/var/lib/postgressql
```
Comment (akash-jain-10) [inline docker/local-metadata/docker-compose-postgres.yml:30]: Can you verify if the Github Actions CI dependent on this compose file will work as expected ?

**Review by akash-jain-10 — CHANGES_REQUESTED** [review]

File: ingestion/src/metadata/cli/docker.py [hunk 4]
```diff
@@ -35,6 +36,13 @@
 logger = cli_logger()
 calc_gb = 1024 * 1024 * 1024
 min_memory_limit = 6 * calc_gb
+# directory = "mysql-data"
+# Parent Directories
+# parent_dir = "/RELEASE/OpenMetadata"
+# Path
+# path = os.path.join(parent_dir, directory)
+# Create the directory
+# os.mkdir(directory)
```
Comment (ayush-shah) [inline ingestion/src/metadata/cli/docker.py:45]: Can we remove the comments around here?

File: ingestion/src/metadata/cli/docker.py [hunk 5]
```diff
@@ -47,7 +55,31 @@
...
+
+    # create a main directory
+    maindir = "docker-volume"
+    if not os.path.exists(maindir):
+        os.mkdir(maindir)
+        db = "db-data"
+        dag_airflow = "ingestion-volume-dag-airflow"
+        dags = "ingestion-volume-dags"
+        tmp = "secrets"
+        om_server = "om-server"
```
Comment (ayush-shah) [inline ingestion/src/metadata/cli/docker.py:68]: Let's make them Global Constants

File: ingestion/src/metadata/cli/docker.py (hunk 4 above)
Comment (ulixius9) [inline ingestion/src/metadata/cli/docker.py:45]: can we clean these comments :)

File: ingestion/src/metadata/cli/docker.py [hunk 6]
```diff
@@ -47,7 +55,31 @@
...
+        final_db_path = os.path.join(maindir, db)
+        final_dag_airflow_path = os.path.join(maindir, dag_airflow)
+        final_dags_path = os.path.join(maindir, dags)
+        final_tmp_path = os.path.join(maindir, tmp)
+        final_om_server = os.path.join(maindir, om_server)
+        os.makedirs(final_db_path, exist_ok=True)
+        os.makedirs(final_dag_airflow_path, exist_ok=True)
+        os.makedirs(final_dags_path, exist_ok=True)
+        os.makedirs(final_tmp_path, exist_ok=True)
+        os.makedirs(final_om_server, exist_ok=True)
```
Comment (ayush-shah) [inline ingestion/src/metadata/cli/docker.py:78]: we can clean this up a little better, let me know if you need any help around here..

we can loop it up, and make directories in the same loop

**Comment by ayush-shah** [comment]
IMO we should remove the created volumes on `metadata docker --clean`
@akash-jain-10 @Vj-L @harshach

File: ingestion/src/metadata/cli/docker.py [hunk 7]
```diff
@@ -47,7 +55,31 @@
...
+
+
 def start_docker(docker, start_time, file_path, ingest_sample_data: bool):
+    docker_volume()
```
Comment (ayush-shah) [inline ingestion/src/metadata/cli/docker.py:82]: let's add a info logger saying that we are creating volume directories

Comment (harshach) [reply ingestion/src/metadata/cli/docker.py:82]: > IMO we should remove the created volumes on `metadata docker --clean`
> @akash-jain-10 @Vj-L @harshach

@ayush-shah we should ask confirmation from the user and delete

**Comment by ayush-shah** [comment]
@Vj-L can you run `make py_format`

**Comment by Vj-L** [comment]
Closing this PR since everything is taken care in other [PR](https://github.com/open-metadata/OpenMetadata/pull/7741)

**Pull request closed without merging**